data/styles/vehicle/include/Icons.mapcss

```diff
@@ -1053,33 +1053,35 @@ area|z16-[amenity=water_point],
 
 /* 7.1 Parking */
 
-node|z15-16[amenity=parking],
-area|z15-16[amenity=parking]
-{icon-image: parking-m.svg;icon-min-distance: 3;z-index: 20000;}
+node|z15-[amenity=parking],
+area|z15-[amenity=parking]
+{icon-image: parking-m.svg;icon-min-distance: 10;z-index: 20000;}
+node|z17-[amenity=parking],
+area|z17-[amenity=parking]
+{icon-min-distance: 15;}
+node|z18-[amenity=parking],
+area|z18-[amenity=parking]
+{icon-min-distance: 25;}
+
 node|z15-16[amenity=parking][access=permissive],
 node|z15-16[amenity=parking][access=private]
-{icon-image: zero-icon.svg; icon-min-distance: 3;z-index: 20000;}
-
-node|z17[amenity=parking],
-area|z17[amenity=parking]
-{icon-image: parking-m.svg;icon-min-distance: 15;}
-node|z17[amenity=parking][access=permissive],
-node|z17[amenity=parking][access=private]
-{icon-image: zero-icon.svg;z-index: 6;icon-min-distance: 7;}
-
-node|z18[amenity=parking],
-area|z18[amenity=parking]
-{icon-image: parking-m.svg;icon-min-distance: 10;}
-node|z18[amenity=parking][access=permissive],
-node|z18[amenity=parking][access=private]
-{icon-image: zero-icon.svg;}
-
-node|z19-[amenity=parking],
-area|z19-[amenity=parking]
-{icon-image: parking-m.svg;icon-min-distance: 6;}
+{icon-image: ; icon-min-distance: 3;z-index: 20000;}
+node|z17-[amenity=parking][access=permissive],
+node|z17-[amenity=parking][access=private]
+{icon-image: ;z-index: 6;icon-min-distance: 7;}
```

Comment:
no point to set z-index or icon-min-distance for an empty icon;

for z18- please use a private parking icon